packages/cli/src/lib/emitUsage.ts

```diff
@@ -29,25 +30,27 @@ export default async function emitUsage(
   numEvents: number,
   numAppMaps: number,
   sampleMetadata?: Metadata
-): Promise<UsageUpdateDto> {
-  const contributors = await Git.contributors(90);
+): Promise<string | undefined> {
   const dto: UsageUpdateDto = {
     events: numEvents,
     appmaps: numAppMaps,
-    contributors: contributors.length,
     metadata: await buildMetadata(appmapDir, sampleMetadata),
     ci: process.env.CI !== undefined,
   };
 
-  Usage.update(dto).catch((e) => {
-    // If this fails, it's not a big deal.
-    // In the future, we may want to cache and aggregate numEvents on
-    // disk and try again at a later date. The client will retry on its
-    // own, so we don't need to do it here.
-    if (verbose()) {
-      console.warn('Failed to update usage', e);
-    }
-  });
+  try {
+    const stats = await stat('.appmap');
+    if (stats.isDirectory()) {
+      const runStatsDirectory = join('.appmap', 'run-stats');
+      await mkdir(runStatsDirectory, { recursive: true });
 
-  return dto;
+      const statsFilePath = join(runStatsDirectory, `${Date.now().toString()}.json`);
```

Comment:
Is this time as number or time as string?

Reply:
`Date.now()` returns the milliseconds since epoch, so a number. The `toString` is probably superfluous given this context.